packages/web-components/@sfx/search-box/src/search-box.ts

```diff
@@ -170,7 +170,14 @@ export default class SearchBox extends Base {
 
   render() {
     return html`
+      <style>
+        sfx-search-box {
+          width: 100%;
+        }
+      </style>
       <input
+        aria-label="${this.placeholder}"
```

Comment:
Is `this.placeholder` adequate for an aria-label? It describes the text inside the search box, but not necessarily what it does. For example, a valid placeholder could be: `...`, which is not descriptive.